layouts/partials/flex/selectnavigation.html

```diff
@@ -50,7 +50,7 @@
   {{end}}
   {{- else}}
     {{- if not .Params.Hidden }}
-      <option value="{{ .RelPermalink}}" {{if eq .URL $currentNode.URL}} selected{{end}}> 
+<option value="{{ .Site.Data.Solo.DocsVersion }}{{ .RelPermalink}}" {{if eq .URL $currentNode.URL}} selected{{end}}> 
```

Comment:
nit: why are we changing the indentation here? i thought it looked better before

Reply:
text editor got confused here, will reformat